realsense2_camera/launch/includes/nodelet.launch.xml

```diff
@@ -88,10 +100,16 @@
   <arg name="initial_reset"            default="false"/>
   <arg name="unite_imu_method"         default="none"/> <!-- Options are: [none, copy, linear_interpolation] -->
   <arg name="allow_no_texture_points"  default="false"/>
-
-
-  <node unless="$(arg external_manager)" pkg="nodelet" type="nodelet" name="$(arg manager)" args="manager" output="screen" required="$(arg required)"/>
-  <node pkg="nodelet" type="nodelet" name="realsense2_camera" args="load realsense2_camera/RealSenseNodeFactory $(arg manager)" required="$(arg required)">
+  <arg name="camera"		           default="camera"/>
+  <arg name="depth_laser_power"	       default="100"/>
+  <arg name="enable_tf_dynamic"        default="true"/>
+  <arg name="depth_exposure"           default="100"/>
+  <arg name="tf_publication_rate"      default="30"/>
```

Comment:
Interesting. At what rate do we publish calibration TFs before/after this change?